vite.config.js

```diff
@@ -16,25 +16,33 @@ export default defineConfig(({mode}) => {
             sourcemap: true,
             rollupOptions: {
                 output: {
-                    assetFileNames: (asset) => asset.name.includes("config.json") ? "assets/[name][extname]": "assets/[name].[hash][extname]",
+                    assetFileNames: (asset) =>
+                        asset.name.includes("config.json") ||
```

Comment:
Is there any way we can put the config file in the root directory rather than in `assets/`? Makes it more obvious for people what to edit when looking for the configuration.

Reply:
I'll do this in a follow up PR (it needs the script that runs after build which is introduced in this PR)

Reply:
done in https://github.com/vector-im/hydrogen-web/pull/735